skyportal/plot.py

```diff
@@ -1448,8 +1497,43 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
 
     smoothed_data = pd.concat(dfs)
 
+    # These values are equivalent from the photometry plot values
```

Comment:
What does this mean?

Reply:
This was here before I got to it but I can definitely improve on it :)